.github/workflows/prod-sync-odfe-cli.yml

```diff
@@ -0,0 +1,31 @@
+name: Prod Sync ODFE CLI Artifacts
+
+on:
+  repository_dispatch:
+    types: [prod-sync-odfe-cli] 
+
+jobs:
+  sync-odfe-cli-artifacts-prod:
+    name: sync-odfe-cli-artifacts-prod
+    runs-on: [ubuntu-18.04]
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Configure AWS Credentials
+      uses: aws-actions/configure-aws-credentials@v1
+      with:
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        aws-region: us-east-1
+
+    - name: S3 sync artifacts
+      run: |
+        #!/bin/bash
+        S3_BUCKET_PROD="artifacts.opendistroforelasticsearch.amazon.com"
+        S3_BUCKET_PLUGIN="artifacts.learning"
+        TAG_VERSION="$(./bin/plugins-info client cutversion --require-install-false | tr ' ' '\n' | uniq | sort -V | head -n 1)"
```

Comment:
`TAG_VERSION="$(./bin/plugins-info client cutversion --require-install-false | tr ' ' '\n' | uniq | sort -V | head -n 1)"`

This line is to grep every plugin versions in plugins.json, not just odfe-cli.
You are mixing sql-cli, odbc, jdbc, etc. together and sort, which is not right imo.

Reply:
Moved to my old code to directly pick the cut version as all the clients require install needs to be false.